_posts/2019-06-23-qsharp-triple-slash.md

```diff
@@ -0,0 +1,62 @@
+# Autogenerating Documentation Comments for Q# in Visual Studio
+
+My name is Joe Clapis, and I've been a professional software engineer for about 10 years. For conventional projects in C#, C++, and
+Python, Visual Studio is my IDE of choice. I like it because it offers a lot of utility that makes it fast and easy to write
+high-quality code. As an example, good code should include extensive documentation for its functions. C# has XML-based documentation
+comments baked into its spec, and Visual Studio can automatically create documentation stubs for you when you type three forward slashes
+above a function:
+
+![alt text](images/qsharptripleslash-1.jpg "C# comment example")

```

Comment:
This is how GitHub flavoured markdown takes alt text, the quotes after only do mouseovers:

```suggestion
![C# comment example](images/qsharptripleslash-1.jpg "C# comment example")
```